chore: remove unnecessary suffix from @vaadin/flow-frontend imports (#131)
vaadin/docs: articles/fusion/application/basics.asciidoc

router.setRoutes(routes);
----

> Now `my-view` is accessible via the root path, that is `http://localhost:8080/`. All the other routes are handled by the server-side router. 

Reviewer: ```suggestion
Now `my-view` is accessible via the root path, that is `http://localhost:8080/`.
All the other routes are handled by the server-side router. 
```

See the <<creating-routes#, Creating routes in TypeScript>> page for more information.

== Hot Reload in Development Mode

When running the application in development mode, all modifications in `frontend` folder are compiled automatically.
Refreshing browser is enough to get the updates.

> NOTE: When adding `index.ts` or `index.html` for the first time, the application server needs to be restarted to update the entry point and the bootstrap template.

Reviewer: ```suggestion
.Server restart is required
[NOTE]
When adding `index.ts` or `index.html`, the application server needs to be restarted to update the entry point and the bootstrap template.
```

I’m not sure the use of admonition/note here is needed. If you feel it is, please add a summarizing title.

I don’t “for the first time” is needed, as you can’t add those files more than once, can you? Or is there more nuance in this, like that if you first add `index.ts` and then restart the server, and if you then add `index.html` server restart is not needed?

Author: very good observation! Thanks for the suggestion, I think it's better.

router.setRoutes(routes);
----

Now `my-view` is accessible via the root path, that is `http://localhost:8080/`.
All the other routes are handled by the server-side router. 
See the <<creating-routes#, Creating routes in TypeScript>> page for more information.

== Hot Reload in Development Mode

When running the application in development mode, all modifications in `frontend` folder are compiled automatically.

> Refreshing browser is enough to get the updates.

Reviewer: ```suggestion
Refreshing the browser is enough to see the updates in the application.
```

.Server restart is required
[NOTE]
When adding `index.ts` or `index.html`, the application server needs to be restarted to update the entry point and the bootstrap template.

== Accessing Backend Data in TypeScript Views

Fusion provides a type-safe and secured way to access data from backend in frontend views using the generated TypeScript code. 

> During the development time, Vaadin scans the backend code and generates TypeScript code which can call corresponding Java methods. 

Reviewer: ```suggestion
Vaadin scans the backend code during development and generates TypeScript code which can call the corresponding Java methods.
```

> The generated code are processed through the same build chain as other TypeScript views. 

Reviewer: ```suggestion
The generated code is processed through the same way as other TypeScript views.
```

> Only necessary code are bundled for your application in production mode. See <<accessing-backend#, Accessing backend from TypeScript>> for more information.

Reviewer: ```suggestion
Only the necessary code is included in the production application bundle. See <<accessing-backend#, Accessing backend from TypeScript>> for more information.
```
